doc/classes/Array.xml

```diff
@@ -307,6 +307,15 @@
 				Searches the array for a value and returns its index or [code]-1[/code] if not found. Optionally, the initial search index can be passed.
 			</description>
 		</method>
+		<method name="find_custom" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="func" type="Callable" />
+			<param index="1" name="from" type="int" default="0" />
+			<description>
+				Searches an array for the first value that causes [param func] to return true, or [code]-1[/code] if not found. Optionally, the initial search index can be passed.
```

Comment:
```suggestion
				Searches the array for the first value that causes [param func] to return true, or [code]-1[/code] if not found. Optionally, the initial search index can be passed.
```
To match other cases, and as this is a member function it makes sense to refer to this, same for other